packages/components/src/pages/install-guide/InvestigateFindings.vue

```diff
@@ -135,7 +135,6 @@ export default {
         maintainability: 0,
       }),
     },
-    findingsEnabled: Boolean,
     analysisEnabled: Boolean,
```

Comment:
What's `analysisEnabled`?

Reply:
`findingsEnabled` is just the setting that users can check or uncheck, `analysisEnabled` indicates if the user is authenticated and has findings enabled.

Reply:
`analysisEnabled` comes from the `AnalysisManager` from [this code](https://github.com/getappmap/vscode-appland/blob/master/src/services/analysisManager.ts#L74-L101)